client/src/components/Navbar/Navbar.stories.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import Navbar from './Navbar';
+import ProfileImage from "../../assets/images/profile_large.png";
 
-storiesOf('Navbar', module)
-  .add("Default", () => (
-    <Navbar />
-  )
-)
+function handleClick() {
+  console.log("clicked");
+}
+
+storiesOf("Navbar", module).add("Default", () => (
+  <Navbar
+    profileImage={ProfileImage}
+    searchClick={handleClick}
+    messageClick={handleClick}
+    burgerClick={handleClick}
```

Comment:
Can you think of more semantic prop names for these?

Reply:
@ZakkMan I felt that they were fairly semantic since they relate to the component that will be activated by them. The burger click could probably be renamed to maybe dropMenuClick. Any suggestions would be great!